docs/framework/wcf/samples/discovery-security-sample.md

```diff
@@ -18,7 +18,7 @@ The Discovery specification does not require that endpoints that participate in
  The secure channel factory creates output or duplex channels that add a compact signature to message headers. To keep messages as small as possible the compact signature format is used. The structure of a compact signature is shown in the following example.  
   
 ```xml  
-<d:Security ... >
+<d:Security>
```

Comment:
I think the ... indicates that there are additional attributes there. Might be best to keep it.

Reply:
ah..ok, I'm modifying this, thank you @gewarren